Mathlib/Geometry/Manifold/Algebra/Monoid.lean

```diff
@@ -410,6 +410,24 @@ theorem contMDiff_finprod_cond (hc : ∀ i, p i → ContMDiff I' I n (f i))
   simp only [← finprod_subtype_eq_finprod_cond]
   exact contMDiff_finprod (fun i => hc i i.2) (hf.comp_injective Subtype.coe_injective)
 
+variable {g : M → G}
+
+theorem ContMDiffWithinAt.pow (hg : ContMDiffWithinAt I' I n g s x) (m : ℕ) :
```

Comment:
Can you add `to_additive` to these theorems?
Can you also add integer powers for groups?